x/rollup/types/keys.go

```diff
@@ -7,9 +7,6 @@ const (
 	// StoreKey defines the primary module store key
 	StoreKey = ModuleName
 
-	// RouterKey is the message route for slashing
-	RouterKey = ModuleName
```

Comment:
Do I read correctly here that all these consts are `"rollup"`? Is this placeholder?